api-report/experimental-tree.api.md

```diff
@@ -652,7 +652,7 @@ export interface OrderedEditSet<TChange = unknown> {
     // (undocumented)
     tryGetEditAtIndex(index: number): Edit<TChange> | undefined;
     // (undocumented)
-    tryGetEditFromIdentifier(editId: EditId): Edit<TChange> | undefined;
+    tryGetEditFromId(editId: EditId): Edit<TChange> | undefined;
```

Comment:
This change looks unrelated to the lint package upgrade.

Reply:
Just made the change! Thank you!